clinica/converters/oasis_to_bids/_converter.py

```diff
@@ -55,6 +55,54 @@ def convert(
             source_dir, destination_dir, subjects=subjects, n_procs=n_procs
         )
         self.convert_clinical_data(clinical_data_dir, destination_dir)
+        self.log_missing_data(source_dir, destination_dir, subjects=subjects)
+
+    def log_missing_data(
+        self, source_dir: Path, bids_dir: Path, subjects: Optional[Path] = None
+    ):
+        """Log missing subjects in BIDS folders.
+
+        Parameters
+        ----------
+        source_dir : Path
+            Path to the OASIS dataset.
+
+        bids_dir : Path
+            Path to the BIDS directory.
+
+        subjects : (Optional) Path
+            Path to the subjects list file.
+            Default=None.
+        """
+        from clinica.converters._utils import compare_bids_ids_lists
+        from clinica.converters.study_models import OASISBIDSSubjectID
+        from clinica.dataset.bids._queries import get_subjects_from_bids_dataset
+        from clinica.utils.stream import cprint
+
+        from ._utils import get_subjects_list
+
+        raw_ids = [
```

Comment:
In other parts of the codebase for converters you can see either `source_id` and `bids_id` to indicate ids under the format of the study and the same ids under the BIDS format, respectively. Since it is not exactly what we have here and to avoid confusion, I suggest we stick with the denomination used inside the function  : 
```suggestion
        expected_ids = [
```